AcadeMeData/test_messages.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from AcadeMeData.models import Messages, User
+
+
+@pytest.fixture
+def user_example():
+    user_data = {'username': "username2212", 'password': "password", 'email': "user@example.com",
+                 'university': "RU",
+                 'degree': "CS"}
+    user = User.create_user(*user_data)
+    return user
+
+
+@pytest.fixture
+def generate_message(user_example, msgID=1, text='This is a test message yo'):
+    user = user_example
+    message = Messages(msgID=msgID, userID=user, text=text)
+    message.save()
+    return message
+
+
+@pytest.mark.django_db
+class TestMessagesModel:
+
+    def test_get_msg(self, generate_message):
+        msg = generate_message
```

Comment:
Please delete it,
the method already called in function declaration
Ho can you search a message? what is the key? 

So calling the fixture. maybe with id will be easier
assume your call to 
def test_get_msg(self, generate_message, msgID=3):

then you can search on line 27
msg_test = Messages.get_msg_by_id(msgID)

Reply:
Yariv I really dont understand you. 
The whole purpose of the test is to check if 2 instances of messages are the same or not
msg = generate_message ---> a variable which holds the fixture
msg_test is a variable that uses get msg by id function to hold a certain message
then after having these 2 vars I assert whether or not they are the same and if the message that came from the function get_msg is an instance of Messages class
BTW this code haven't been changed and you already approved it so I really dont understand the comments about an already seen and checked code.
I dont see nothing thats need to be changed here. 
(Some will say msg = generate_msg is not needed but its a matter of style..)